tlf/handle_extension.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
```

Comment:
Sigh, I don't know why this isn't getting picked up as a rename. But I didn't change anything substantial here.